presto-session-property-managers/src/test/java/io/prestosql/plugin/session/db/TestingMySqlContainer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.session.db;
+
+import org.testcontainers.containers.MySQLContainer;
+
+public class TestingMySqlContainer
+        extends MySQLContainer<TestingMySqlContainer>
+{
+    public TestingMySqlContainer()
+    {
+        super("mysql:8.0.12");
+    }
+
+    @Override
+    public String getJdbcUrl()
+    {
+        return super.getJdbcUrl() + "?useSSL=false&allowPublicKeyRetrieval=true";
```

Comment:
could you use `constructUrlForConnection`?

Reply:
Good point, this is possible now since it’s a subclass.